src/components/LandingPage/LandingPageCallToAction/index.tsx

```diff
@@ -5,7 +5,7 @@ export const LandingPageCallToAction = (): JSX.Element => {
     return (
         <div className="ctas flex flex-col items-center justify-center w-full max-w-xl sm:mx-auto sm:flex-row">
             <CallToAction icon="rocket" type="primary" to="/sign-up?src=home-hero">
-                Get Started
+                Deploy Now
```

Comment:
Should we leave this as `Get started` for now since we have the test which leads to a page where you _then_ decide if you're deploying or signing up for Cloud?